Engineering/QA/Boundary and Equivalence Partitioning Testing

```diff
@@ -0,0 +1,74 @@
+---
+Tags: Engineering/QA, Testing Type, QA Terms
+Author: Binh Le
+Date: 2022-10-15
+---
```

Comment:
The keys frontmatter/metadata follow a YAML format and are lower-case. Check the our docs in case there are any confusions: https://github.com/dwarvesf/brain/blob/master/CONTRIBUTING.md

Reply:
Updated